src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml

```diff
@@ -1668,18 +1668,7 @@ let snark_hashes =
       let json = Cli_lib.Flag.json in
       let print = Core.printf "%s\n%!" in
       fun () ->
-        let hashes =
```

Comment:
Why keep a command that only prints `[]`/`<empty>`?

Even is we want to keep it, let's simplify the code down to:

```
if json then Core.print "[]\n"
```